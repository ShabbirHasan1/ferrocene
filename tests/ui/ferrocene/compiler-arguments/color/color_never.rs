// check-fail
// compile-flags: --error-format human --color never

fn main() {
    let x = x; //~ ERROR cannot find value `x` in this scope
}

// ferrocene-annotations: um_rustc_color
// ferrocene-annotations: um_rustc_error_format
