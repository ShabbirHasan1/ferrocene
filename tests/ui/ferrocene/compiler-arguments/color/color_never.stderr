error[E0425]: cannot find value `x` in this scope
  --> $DIR/color_never.rs:5:13
   |
LL |     let x = x;
   |             ^ not found in this scope

error: aborting due to previous error

For more information about this error, try `rustc --explain E0425`.
